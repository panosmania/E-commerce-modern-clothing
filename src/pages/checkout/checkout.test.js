import { shallow } from "enzyme";
import React from "react";
import toJson from "enzyme-to-json";

import { CheckoutPage } from "./checkout.component";

describe("CheckoutPage component", () => {
  let wrapper;
  beforeEach(() => {
    const mockProps = {
      cartItems: [],
      total: 100,
    };

    wrapper = shallow(<CheckoutPage {...mockProps} />);
  });

  it("should render CheckoutPage component", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
