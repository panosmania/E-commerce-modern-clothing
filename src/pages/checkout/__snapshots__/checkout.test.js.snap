// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutPage component should render CheckoutPage component 1`] = `
<div
  className="checkout-page"
>
  <div
    className="checkout-header"
  >
    <div
      className="header-block"
    >
      <span>
        Product
      </span>
    </div>
    <div
      className="header-block"
    >
      <span>
        Description
      </span>
    </div>
    <div
      className="header-block"
    >
      <span>
        Quantity
      </span>
    </div>
    <div
      className="header-block"
    >
      <span>
        Price
      </span>
    </div>
    <div
      className="header-block"
    >
      <span>
        Remove
      </span>
    </div>
  </div>
  <div
    className="total"
  >
    <span>
      TOTAL: $
      100
    </span>
  </div>
  <div
    className="test-warning"
  >
    *Please use the following test credict card for payments*
    <br />
    4242 4242 4242 4242 - Exp: 01/30 - CVV: 123
  </div>
  <StripeCheckoutButton
    price={100}
  />
</div>
`;
